server/xml.server.php

```diff
@@ -34,7 +34,7 @@
 
 /* Set the correct headers */
 header("Content-type: text/xml; charset=" . AmpConfig::get('site_charset'));
-header("Content-Disposition: attachment; filename=information.xml");
+// header("Content-Disposition: attachment; filename=information.xml");
```

Comment:
Ya, I'm not sure of it's purpose. It might have one. Better safe than sorry I guess?